sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -455,6 +466,23 @@ public Builder sharePidNamespace(Boolean sharePidNamespace) {
             return this;
         }
 
+        /**
+         * Sets the {@code host volumes} and returns a reference to this Builder so that the methods can be
+         * chained together.
+         *
+         * @param hostVolumes the {@code hostVolumes} to set
+         * @return a reference to this Builder
+         */
+        public Builder hostVolumes(Collection<HostVolumeSpec> hostVolumes) {
```

Comment:
This could just be `this.hostVolumes = (hostVolumes != null) ? hostVolumes : new ArrayList<>();`